evm/vm/forks/frontier/headers.py

```diff
@@ -4,14 +4,18 @@
     validate_gt,
     validate_header_params_for_configuration,
 )
+
 from evm.constants import (
     GENESIS_GAS_LIMIT,
     DIFFICULTY_ADJUSTMENT_DENOMINATOR,
     DIFFICULTY_MINIMUM,
     BOMB_EXPONENTIAL_PERIOD,
     BOMB_EXPONENTIAL_FREE_PERIODS,
-    FRONTIER_DIFFICULTY_ADJUSTMENT_CUTOFF,
 )
+from evm.vm.forks.frontier.constants import (
```

Comment:
I think for these imports within the `frontier` module importing another module from within the `frontier` module it may be more legible to use a relative import:

```
from .constants import (
    ...
)
```

Same applies across the other forks when they import from their own `constants` module.